web-client/public/js/setuphandlers.js

```diff
@@ -0,0 +1,28 @@
+/* eslint no-unused-vars: [2, {"varsIgnorePattern": "updateApp|grnState"}] */
+var updateApp = require("./updateapp");
+export var setupHandlers = function (grnState) {
+
+    var GREY_EDGES_DASHED_MENU = "#grey-edges-dashed-menu";
+    var GREY_EDGES_DASHED_SIDEBAR = "#dashedGrayLineButton";
+
+    if (!$(GREY_EDGES_DASHED_MENU).prop("checked")) {
+        grnState.dashedLine = false;
+    } else {
+        grnState.dashedLine = true;
+    }
+
+    if ($(GREY_EDGES_DASHED_SIDEBAR).prop("checked")) {
+        grnState.dashedLine = true;
+    } else {
+        grnState.dashedLine = false;
+    }
```

Comment:
These `if` blocks shouldn't be here. Remember, `grnState` is the _single source of truth_. This code is going backwards: it's setting `grnState` based on the state of the UI. _This should not happen._ Instead, `grnState` _determines_ the state of the UI. That's why we call `updateApp` at the very beginning.

Reply:
I meant to make these 3 equal signs, would that be more correct?